### PR TITLE
Tests for package addition and deletion

### PR DESCRIPTION
**Pull request by anand0707**
Tests creates, verify and deletes a package.

We are using LiveServerTestCase , using localhost user and repo .Since Django LiveServerTestCase cannot connect external urls test is creating local repo and series  before verifying package addition  and deletion.

### CODE REVIEW

File: aasemble/django/apps/buildsvc/tests_functional.py [hunk 1]
```diff
@@ -47,3 +49,109 @@ def test_secured_pages_open_after_login(self):
...
+        git_url = "https://github.com/aaSemble/python-aasemble.django.git"
+        self.create_new_package_source(git_url=git_url, branch='master', series='myrepo/myseries')
+        self.assertEqual(self.verify_package_source(git_url=git_url), True, 'Package not created')
+        self.delete_package_source()
+        self.assertEqual(self.verify_package_source(git_url=git_url), False, 'Package not deleted')
+        # We will follow opposite order as that of creation
+        delete_series(name='myseries')
+        delete_repo(name='myrepo')
+        delete_group(name='mygrp')
+        delete_user(username='myuser')
```
Comment (sorenisanerd) [inline aasemble/django/apps/buildsvc/tests_functional.py]: These clean-up steps are unnecessary. The test infrastructure should make sure each tests starts with a clean database, so each one does not need to clean up.

File: aasemble/django/apps/buildsvc/tests_functional.py [hunk 2]
```diff
@@ -47,3 +49,109 @@ def test_secured_pages_open_after_login(self):
...
+              user creation.
+           2. Create a group. This will serve as 'Extra admin' for test repo.
+           3. Create a test repo under user that we have created in step 1
+           4. Create a series with the repo of step 3.
+           5. Try to create a package.
+           6. Verify if the package has been created.
+           7. Try to delete the package
+           8. Verify if the package has been deleted
+           9. Peform cleanup like delete user. repo etc that we have created.'''
+        session_cookie = create_session_cookie(username='myuser', password='123456')
```
Comment (sorenisanerd) [inline aasemble/django/apps/buildsvc/tests_functional.py]: Now that the fixtures contain a number of users, let's not add more tests that create their own users.

In other words: Please use one of the existing users defined in aasemble/django/apps/api/fixtures/complete.json. See  docs/Testing.md for a description of each user.

File: aasemble/django/tests.py [hunk 3]
```diff
@@ -107,3 +108,50 @@ def test_run_cmd_alternate_stdout(self):
...
+
+
+def delete_group(name):
+    group = Group.objects.get(name=name)
+    group.delete()
+
+
+def delete_user(username):
+    user = User.objects.get(username=username)
+    user.delete()
```
Comment (sorenisanerd) [inline aasemble/django/tests.py]: All of these methods are unnecessary. The fixtures provide groups, repos and series and the test infrastructure will perform the necessary cleanup.

**Comment by anand0707** [comment]
Changed the code as per suggestions.

Now we are using user 'Dennis'.

**Pull request merged**